src/lib/storage/index/adaptive_radix_tree/adaptive_radix_tree_index.cpp

```diff
@@ -16,6 +16,12 @@
 
 namespace opossum {
 
+size_t AdaptiveRadixTreeIndex::estimate_memory_consumption(ChunkOffset row_count, ChunkOffset distinct_count,
+                                                           uint32_t value_bytes) {
+  // ToDo(anyone): If you use this index in combination with the Tuning subsystem, you need to properly implement this.
```

Comment:
What is this todo good for? The fact that it's not implemented is obvious, the explanation of when you might need it feels redundant.